conf/model.go

```diff
@@ -76,3 +64,7 @@ type NetworkConfig struct {
 
 	Tag string
 }
+
+type NodeConfig interface {
+	ToOverrideEnv(nc NetworkConfig) []byte
+}
```

Comment:
i'm still on the fence on this interface pattern, may be better served as a utility function